src/ui/public/courier/__tests__/saved_object.js

```diff
@@ -434,10 +428,12 @@ describe('Saved Object', function () {
         };
 
         stubESResponse({
-          _id: indexPatternId,
-          _type: 'dashboard',
-          _source: {},
-          found: true
+          id: indexPatternId,
+          type: 'dashboard',
+          attributes: {
+            title: 'testIndexPattern'
+          },
+          _version: 2
```

Comment:
It's actually _version since it comes from SavedObject - we can revisit later if that's ok.